NetKAN/FirespitterCore.netkan

```diff
@@ -1,5 +1,5 @@
 {
-    "spec_version"    : "v1.2",
+    "spec_version"    : "v1.10",
```

Comment:
should be 1.16 according the [install-spec](https://github.com/KSP-CKAN/CKAN/blob/master/Spec.md#install)